bundle/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Executor.scala

```diff
@@ -0,0 +1,49 @@
+package edu.gemini.seqexec.server
+
+import edu.gemini.seqexec.server.SeqexecFailure._
+import edu.gemini.spModel.config2.{Config, ConfigSequence, ItemKey}
+import edu.gemini.spModel.obscomp.InstConstants.INSTRUMENT_NAME_PROP
+import edu.gemini.spModel.seqcomp.SeqConfigNames.INSTRUMENT_KEY
+
+import scalaz._
+import Scalaz._
+
+import scalaz.concurrent.Task
+
+/**
+ * Created by jluhrs on 4/28/15.
+ */
+final class Executor(sequenceConfig: ConfigSequence) {
+
+  def run: \/[Throwable, List[StepResult]] = runSequence(sequenceConfig.getAllSteps.toList).attemptRun
+
+  private def runSequence(config: List[Config]): Task[List[StepResult]] = {
+    (config map (runStep(_))).sequenceU
```

Comment:
Just a minor style thing.  I believe this can be

```scala
private def runSequence(config: List[Config]): Task[List[StepResult]] =
  config.map(runStep).sequenceU
````

since `runStep` is a function that takes a single `Config` argument.  I understand the style `a map foo` (as opposed to `a.map(foo)`) is not encouraged except maybe for some DSLs.